ui/src/preview/preview-view.tsx

```diff
@@ -225,41 +288,20 @@ const pendingCandidate = <T extends PublishCandidate>(candidate: T) => ({
     pendingValidation: true,
 });
 
-const singleRowPublishRequestOfPreviewTableEntry = (
```

Comment:
Eivät enää tarpeellisia, oikeanlainen tietorakenne (PublishRequestIds) päätyy nyt jo lähteeltä minkä vuoksi näitä funkkareita edes tarvittiin.